### ISSUE TITLE
PyTorch runs INCREDIBLY SLOW

### ISSUE THREAD

**Issue by dlmpal** [pytorch/pytorch#65597]
Hi , first time reporting an issue , so my format may be a bit off.  Im transitioning from tensrflow to torch , and for some reason 
my lstm implementation seems to be very slow , compared to the tensorflow counterpart. Im running it on the CPU , and yes even though CPU usage is at 100% (num threads seems not to be the issue), the model training is painfully slow. 

Im also presenting my code below. Any input is highly appreciated!!

### CODE ###
class RNN(nn.Module):
    
    def __init__(self):
        super(RNN,self).__init__()   
        self.lstm1  = nn.LSTM(batch_first = True , hidden_size = 64 , input_size = 4 , num_layers = 3)
        self.dense1 = nn.Linear(in_features = 64 , out_features = 64)
        self.dense2 = nn.Linear(in_features = 64 , out_features = 4)
        
    def forward(self , X ):
       X , _ = self.lstm1(X)
       X     = X[:,-1,:]
       X     = self.dense1(X)
       X     = nn.functional.relu(X)
       X     = self.dense2(X)
       return X 
   

model = RNN()
model.double()

loss_fn = nn.MSELoss()
optimizer = torch.optim.Adam(lr = 0.1 , params = model.parameters())
epochs = 150 

for epoch in range(epochs):
    
    for batch , (seq , target) in enumerate(trainset_loader):
        pred = model(seq)
        loss = loss_fn(pred , target)
        optimizer.zero_grad()
        loss.backward()
        optimizer.step()
        
        print(f"loss : {loss.item()} -- epoch : {epoch} -- batch -- {batch}")

**Comment by gchanan** [comment]
If you are looking for debugging help, please use the discussion forums: https://discuss.pytorch.org/.
